doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -102,9 +103,17 @@ Complete the following steps to configure the modem trace level at compile time:
 The application can use the :c:func:`nrf_modem_lib_trace_level_set` function to set the desired trace level.
 Passing ``NRF_MODEM_LIB_TRACE_LEVEL_OFF`` to the :c:func:`nrf_modem_lib_trace_level_set` function disables trace output.
 
+.. note::
+   The modem stores the current trace level on passing the ``AT+CFUN=0`` command.
+   If the trace level stored in the modem is ``NRF_MODEM_LIB_TRACE_LEVEL_OFF``, the application must enable traces in the modem using the :c:func:`nrf_modem_lib_trace_level_set` function or by enabling any of the Kconfig options mentioned in step 2.
```

Comment:
To refer a step, we normally provide in this manner. For example, https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_dev/wifi_coex/index.html#enabling-wi-fi-coexistence.
But in this case we could add this label after line no:96: 
.. _trace_level_options:
and then here, add the following suggestion, so that it is linked to those options:
```suggestion
   If the trace level stored in the modem is ``NRF_MODEM_LIB_TRACE_LEVEL_OFF``, the application must enable traces in the modem using the :c:func:`nrf_modem_lib_trace_level_set` function or by enabling any of the :ref:`aforementioned Kconfig options <trace_level_options>`.
```

Reply:
ok, I've updated the doc with the suggestion.